### ISSUE TITLE
Getting Runtime Exception with Regular Use

### ISSUE THREAD

**Issue by Spittal** [florent37/DaVinci#4]
```
java.lang.RuntimeException: An error occured while executing doInBackground()
            at android.os.AsyncTask$3.done(AsyncTask.java:300)
            at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
            at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
            at java.util.concurrent.FutureTask.run(FutureTask.java:242)
            at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
            at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
            at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
            at java.lang.Thread.run(Thread.java:818)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'android.graphics.Bitmap com.github.florent37.davinci.DiskLruImageCache.getBitmap(java.lang.String)' on a null object reference
            at com.github.florent37.davinci.DaVinci.loadFromDiskLruCache(DaVinci.java:475)
            at com.github.florent37.davinci.DaVinci.loadFromLruCache(DaVinci.java:461)
            at com.github.florent37.davinci.DaVinci.loadImage(DaVinci.java:359)
            at com.github.florent37.davinci.DaVinci.access$000(DaVinci.java:44)
            at com.github.florent37.davinci.DaVinci$1.doInBackground(DaVinci.java:183)
            at com.github.florent37.davinci.DaVinci$1.doInBackground(DaVinci.java:180)
            at android.os.AsyncTask$2.call(AsyncTask.java:288)
            at java.util.concurrent.FutureTask.run(FutureTask.java:237)
            at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
            at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
            at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
            at java.lang.Thread.run(Thread.java:818)
```

Using the simple code in a fragment on the watch

```
ImageView logoView = (ImageView) mRootView.findViewById(R.id.brewery_logo);
DaVinci.with(getActivity()).load("/image/0").into(logoView);
```

I'm curious if maybe I'm not including this correctly? Any help would be an incredible help!

Thanks!

**Comment by florent37** [comment]
Ok I made an error on the DaVinci 1.1 build.
I moved the DiskLruCache initialisation to clear() for test and I didn't see it because in my activity I called clear().

Thanks for reporting this issue, I fixed it and applied a patch to the 1.1

**Comment by Spittal** [comment]
The 1.1 build is not available to be compiled by gradle right now, correct?

example:

```
compile ('com.github.florent37:davincidaemon:1.1.0@aar'){
        transitive = true
    }
```

**Comment by florent37** [comment]
I fixed it in the 1.0.1 directly

``` java
compile ('com.github.florent37:davincidaemon:1.0.1@aar'){
        transitive = true
    }
```

**Comment by florent37** [comment]
if not working, try to clear your cache 

``` shell
rm -rf $HOME/.gradle/caches/modules-2/files-2.1/com.github.florent37/
```
